src/devices/bus/sg1000_exp/sp400.cpp

```diff
@@ -0,0 +1,249 @@
+// license: BSD-3-Clause
+// copyright-holders: Charles MacDonald, Devin Hill, Fabio Dalla Libera
```

Comment:
How much actual code that qualifies as a creative work by any of these people is left?  There’s very little left at all that isn’t “facts” (address maps, ROM definitions) where only the formatting is eligible for copyright protection.  I’d also hazard a guess that Charles MacDonald didn’t contribute much to the actual code in the first place.

Reply:
ok, I have reduced it to just fabiodl.  I have sent him an email asking if that is ok with him.